### PR TITLE
fix: stories tabs to toggle button

### PR DESCRIPTION
**Pull request by eljefe223**
## New Behavior

Aligns more closely to storybook 8 api and fixes broken args in stories

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/accordion-item/accordion-item.stories.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { html } from '@microsoft/fast-element';
```
Comment (fabricteam) [inline packages/web-components/src/accordion-item/accordion-item.stories.ts:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32938&commitId=7872cc72ae6bc5500c899dbc25f35e12dfd8455b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/web-components/src/avatar/avatar.stories.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { html } from '@microsoft/fast-element';
-import { type NewMeta as Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
+import { Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
```
Comment (radium-v) [inline packages/web-components/src/avatar/avatar.stories.ts:2]: `Meta` is still a type (it should be updated in all stories in this PR)
 
```suggestion
import { type Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
```

File: packages/web-components/src/tablist/tablist.stories.ts [hunk 3]
```diff
@@ -1,211 +1,150 @@
...
-      orientation=${x => x.orientation}
-      appearance=${x => x.appearance}
-      ?disabled=${x => x.disabled}
-      size=${x => x.size}
-      activeid=${x => x.activeid}
-      @change="${() => changeTab()}"
+      orientation=${story => story.orientation}
+      appearance=${story => story.appearance}
+      ?disabled=${story => story.disabled}
+      size=${story => story.size}
```
Comment (radium-v) [inline packages/web-components/src/tablist/tablist.stories.ts:14]:
```suggestion
      orientation="${story => story.orientation}"
      appearance="${story => story.appearance}"
      ?disabled="${story => story.disabled}"
      size="${story => story.size}"
```

File: packages/web-components/src/toggle-button/toggle-button.stories.ts [hunk 4]
```diff
@@ -1,128 +1,210 @@
...
 
-export const Appearance = renderComponent(html<ToggleButtonStoryArgs>`
-  <fluent-toggle-button>Default</fluent-toggle-button>
-  <fluent-toggle-button appearance="primary">Primary</fluent-toggle-button>
-  <fluent-toggle-button appearance="outline">Outline</fluent-toggle-button>
-  <fluent-toggle-button appearance="subtle">Subtle</fluent-toggle-button>
-  <fluent-toggle-button appearance="transparent">Transparent</fluent-toggle-button>
-`);
+export const Appearance: Story = {
+  render: renderComponent(html`<div>${repeat(story => story.storyItems, storyTemplate)}</div> `),
```
Comment (radium-v) [inline packages/web-components/src/toggle-button/toggle-button.stories.ts:171]: nit: the wrapping `<div>`s probably aren't necessary

Comment (eljefe223) [reply packages/web-components/src/toggle-button/toggle-button.stories.ts:171]: In pre storybook 8 it is, othwerwise the code preview does not work.

File: packages/web-components/src/text/text.stories.ts [hunk 5]
```diff
@@ -38,184 +36,239 @@ const generateSemanticElementTemplate = (as: string, content: string) => {
...
+    align=${story => story.align}
+    font=${story => story.font}
+    size=${story => story.size}
+    weight=${story => story.weight}
+    ?nowrap=${story => story.nowrap}
+    ?truncate=${story => story.truncate}
+    ?italic=${story => story.italic}
+    ?underline=${story => story.underline}
+    ?strikethrough=${story => story.strikethrough}
+    ?block=${story => story.block}
```
Comment (radium-v) [inline packages/web-components/src/text/text.stories.ts:50]:
```suggestion
    align="${story => story.align}"
    font="${story => story.font}"
    size="${story => story.size}"
    weight="${story => story.weight}"
    ?nowrap="${story => story.nowrap}"
    ?truncate="${story => story.truncate}"
    ?italic="${story => story.italic}"
    ?underline="${story => story.underline}"
    ?strikethrough="${story => story.strikethrough}"
    ?block="${story => story.block}"
```

**Review by radium-v — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Pull request merged**